DOCUMENTATION.md

```diff
@@ -172,25 +172,6 @@ let personHandler =
         }
 ```
 
-The `task {}` CE is an independent project maintained by [Robert Peele](https://github.com/rspeele) and can be used from any other F# application as well. All you have to do is add a reference to the `TaskBuilder.fs` NuGet library and open the `FSharp.Control.Tasks.V2` module:
-
-```fsharp
-open FSharp.Control.Tasks.V2
-
-let readFileAndDoSomething (filePath : string) =
-    task {
-        use stream = new FileStream(filePath, FileMode.Open)
-        use reader = new StreamReader(stream)
-        let! contents = reader.ReadToEndAsync()
-
-        // do something with contents
-
-        return contents
-    }
-```
-
-For more information please visit the official [TaskBuilder.fs](https://github.com/rspeele/TaskBuilder.fs) GitHub repository.
-
```

Comment:
How about leaving this bit in, just in case the reader does want to know more:

```
The `task {}` CE is an independent project maintained by [Robert Peele](https://github.com/rspeele), for more information please visit the official [TaskBuilder.fs](https://github.com/rspeele/TaskBuilder.fs) GitHub repository.
```

I know that opening part is duplicating what is above a bit (i.e. task{} is not from Giraffe), but it's worth reiterating, and it gives credit to the creator.